prompts/litellm/google-gemini-research.yml

```
@@ -0,0 +1,46 @@
+title: Google Gemini 2.5 Pro Research Assistant  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)